curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md

```diff
@@ -73,7 +73,7 @@ assert(/myStr\s*=/.test(code));
 ## --seed-contents--
 
 ```js
-const myStr = ""; // Change this line
+let myStr; // Change this line
```

Comment:
I worry we will get the usual complaints about the instructions being in the test text, and not in the instruction body:
> `myStr` should be created using the `const` keyword.

Reply:
Awesome, thanks for catching that, @ShaunSHamilton. I'll go back in and add that instruction to the instruction text.

Reply:
I vote for number 2

Reply:
@scissorsneedfoodtoo Both of those are valid solutions. I am leaning towards option **1**, because it is less wordy. Although, I might even leave the `const` test in there, because Campers have a habit of changing more than they are asked.

Reply:
Awesome, thanks for the input @ieahleen and @ShaunSHamilton.

I'm also leaning toward option 1 because it's a bit closer to what we have now. The only issue with leaving the `const` test is that it will pass with just the seed code. This was brought up in an earlier review.

I'm fine with it, as the current version of the challenge works this way with the `var` test. But I could go either way with options 1 or 2.

Reply:
I went ahead with option 1 because it's closer to what we currently have.

If anyone else chimes in and would rather go with option 2, I can go back in and change the seed and wording.

Otherwise, I think this is ready for another quick review of the "Concatenating Strings with Plus Operator" challenge and for the steps Oliver mentioned in this comment above: https://github.com/freeCodeCamp/freeCodeCamp/pull/43133#issuecomment-904411754